samples/machine-learning/parallel-half-moons/data.json

```diff
@@ -0,0 +1,774 @@
+{
```

Comment:
Is the data the same as for the other sample? If so, does the duplication make sense?

Reply:
I could remove it from `parallel-half-moons` and change the path used to look for it. The only issue would be with onboarding to the Samples Browser, which expects that all files used by a single sample live in the same directory.

Reply:
Ah ok, never mind then.